client/src/components/Header.tsx

```diff
@@ -15,40 +16,40 @@ import LogoutIcon from '@mui/icons-material/Logout';
 import { FC, MouseEvent, useCallback, useState } from 'react';
 import { useNavigate } from 'react-router-dom';
 import {
+  ADMIN_PAGE,
   HOME_PAGE,
-  POST_CREATE,
+  MY_POSTS_PAGE,
   PROFILE_PAGE,
-  TAGS_PAGE,
-  USERS_PAGE,
+  SUBSCRIPTIONS_PAGE,
 } from '../utils/constants';
 import { useAppDispatch, useAppSelector } from '../store/hooks';
 import { signOut } from '../store/auth/SignOut';
 
 const pages = [
   {
-    name: 'All News',
+    name: 'Home',
     link: HOME_PAGE,
     access: ['reader', 'writer', 'admin'],
```

Comment:
Please separate access levels to enum to avoid strings repetations. And below

Reply:
Array "access" moved to constants file and declared with enums UserRole